form.php

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css"/>
+    <meta name="viewport" content="width=device-width">
+    <title>Cats of the World</title>
+    <link href="https://fonts.googleapis.com/css2?family=Bungee+Shade&family=Roboto+Mono:wght@300&family=Work+Sans&display=swap" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="styles.css" type="text/css"/>
+</head>
+<body>
+<header>
+    <h1>Cats of the World! 	&#127758</h1>
+</header>
+<main class="formMain">
+    <form>
```

Comment:
Should you set the action and method properties here? It's currently displaying my inputted data in the URL